.github/issue_template.md

```diff
@@ -14,6 +14,14 @@ The output of `jx version` is:
 COPY OUTPUT HERE
 ```
 
+### Diagnostic information
+
+The output of `jx diagnose --show=version` is:
```

Comment:
does this require #7192 to be ready and merged yet? 
Right now I get:

```
jx diagnose --show=version
Running in namespace: jx
ERROR: Unable to get the Jenkins X Version
error: Command failed 'jx version --no-version-check': Error: unknown flag: --no-version-check


Options:
      --short=false: Print version
Usage:
  jx version [flags] [options]
Use "jx options" for a list of global command-line options (applies to all commands). exit status 1
```

Reply:
Ah, I see the error, I removed the `--no-version-check`, and it complains about that. I am going to add those flags I removed, back to the version command, and deprecate them. Also is there another command which does a version check? If not, I need to write one as part of this PR. I dont think `jx version` should really check if there are newer versions of jx, wdyt? Actually, should we even do a version check?

Reply:
> Actually, should we even do a version check?

No, I don't think this is needed.

Reply:
Should the command in the template just be `jx diagnose`? For `jx diagnose --show=version` I get:

```
jx diagnose --show=version
Flag --show has been deprecated, use jx diagnose <object> instead. This will be removed on July 1, 2020
Running in namespace: jx
NAME                          VERSION
jx                            2.1.60-dev+83850f2b5
Kubernetes cluster            v1.14.10-gke.36
kubectl (installed in JX_BIN) v1.13.2
helm client                   2.12.2
git                           2.20.1
Operating System              Mac OS X 10.14.6 build 18G4032
Error: unknown flag: --show
Usage:
  version [flags]

Flags:
  -h, --help    help for version
      --short   Print version

error: unknown flag: --show
```